### ISSUE TITLE
[Bug]: Client crashes with large book with multiple files

### ISSUE THREAD

**Issue by ljlongwing** [advplyr/audiobookshelf-app#1319]
### What was the Problem?

I was recently experiencing https://github.com/advplyr/audiobookshelf-app/issues/1318, and decided that the app might work better with chapter size files instead of one big monolithic file....the book is 20 hours long with 70 chapters, a little over 1.8 gig in size....I split the book into chapters, so I now have 70 files covering the same amount of time.  I thought the many smaller files would fix the issues I was having, and it seemed to, but it caused another more troublesome problem.  My client started crashing when it was approaching a chapter change.  The phone would noticeably slow down, the audio would start to jitter...and then end completely.  I would then need to re-launch the program, hit play again, and leave the program in the foreground till it was able to transition to the new chapter, then I could go back to my other apps while listening, till it needed to chapter again.

### Steps to Reproduce the Issue

I can provide the book with all files to show the issue if needed

### What was Expected?

I wouldn't expect to need to 'babysit' the app during chapter changes.

### Phone Model

Samsung Galaxy S22+

### Phone OS

Android 14

### Audiobookshelf App Version

Android App - 0.9.76

### Installation Source

Google Play Store

**Comment by nichwall** [comment]
Duplicate of https://github.com/advplyr/audiobookshelf-app/issues/1167?

**Comment by ljlongwing** [comment]
> Duplicate of #1167?

No, this bug is about playback during book that is multiple files, that defect is about crashing while downloading a large file.

**Comment by kajuberdut** [comment]
This is easy to reproduce with the publicly available and free HPMOR podcast in 6 part m4b book format: https://hpmorpodcast.com/?page_id=56

Simply download that and import it into audiobookshelf, the web player crashes the full chrome tab that it is in. I'm using a modern 4 core intel box with 16gb memory for my server and an i7/24gb laptop for browser/web-player, definitely not something that should crash on an audio file.
